airflow/bin/cli.py

```diff
@@ -934,7 +935,7 @@ def connections(args):
     if args.list:
         # Check that no other flags were passed to the command
         invalid_args = list()
-        for arg in ['conn_id', 'conn_uri', 'conn_extra']:
+        for arg in ['conn_id', 'conn_uri', 'conn_type', 'conn_host', 'conn_login', 'conn_password', 'conn_schema', 'conn_port', 'conn_extra']:
```

Comment:
can we clean up the code a bit for better readability? 
e.g., alternative_conn_specs = ['conn_type', 'conn_host', 'conn_login', 'conn_password', 'conn_schema', 'conn_port'] and use alternative_conn_specs where it makes sense.